docs/0.6/references/cli/splinter-authid-update.1.md

```diff
@@ -113,4 +113,4 @@ SEE ALSO
 | `splinter-authid-list(1)`
 | `splinter-authid-show(1)`
 |
-| Splinter documentation: https://www.splinter.dev/docs/0.5/
+| Splinter documentation: https://www.splinter.dev/docs/0.6/
```

Comment:
We will have to update these in the actually man pages as well https://github.com/Cargill/splinter/tree/0-6/cli/man as well but I can take care of that.